revolv/static/project.scss

```diff
@@ -43,6 +43,439 @@
     }
 }
 
+body {
+    font-family: Lato;
+}
+
+.row {
+    margin-right: 0px;
+    margin-left: 0px;
+    max-width: 100rem;
+
+    &.heading {
+        height: 500px;
+    }
+
+    &.about, &.project-updates, &.dashboard {
+        margin-right: 15%;
+        margin-left: 15%;
+        max-width: 70%;
+    }
+
+    &.dashboard {
+        height: 600px;
+    }
+
+    &.categories {
+        max-width: 100%;
+    }
+}
+
+.header-img-div {
+    width: 100%;
+    height: 500px;
+    position: absolute;
+    background-size: cover;
+
+    .overlay {
+        width: 100%;
+        height: 100%;
+        top: 0;
+        left: 0;
+        background: rgba(0, 0, 0, .5);
+        position: absolute;
+        
+        .donate-button-header {
+            border-color: white;
+            border-width: 2px;
+            border-style: solid;
+            width: 15%;
+            height: 10%;
+            top: 410px;
+            left: 80%;
+            z-index: 100;
+            position: absolute;
+            transition: all 0.3s;
+
+            .donate-link {
+                width: 100%;
+                height: 100%;
+                display: inline-block;
+                
+                p {
```

Comment:
If you actually look in the web inspector, this element overflows the button on the bottom. I think it's because it had a line-height that is set elsewhere maybe?